controllers/actions.github.com/autoscalingrunnerset_controller.go

```diff
@@ -57,6 +57,7 @@ type AutoscalingRunnerSetReconciler struct {
 	ControllerNamespace                           string
 	DefaultRunnerScaleSetListenerImage            string
 	DefaultRunnerScaleSetListenerImagePullSecrets []string
+	UpdateStrategy                                string
```

Comment:
It would be awesome if we create a type for update strategy.
```go
type UpdateStrategy string
// Update strategies
const (
        // UpdateStrategyImmediate will not recreate resources until all
        // pending / running jobs have completed.
        // This can lead to a larger time to apply the change but it will ensure
        // that you don't have any overprovisioning of runners.
        UpdateStrategyImmediate = UpdateStrategy("immediate")
        // docs for update strategy
        UpdateStrategyEventual = UpdateStrategy("eventual")
)
```

And then the UpdateStrategy fields is of type `UpdateStrategy`. We can document the strategies on package level